cmd/dlt/machinepool/options.go

```diff
@@ -0,0 +1,44 @@
+package machinepool
+
+import (
+	"fmt"
+
+	"github.com/openshift/rosa/pkg/machinepool"
+	"github.com/openshift/rosa/pkg/reporter"
+)
+
+type DeleteMachinepoolUserOptions struct {
```

Comment:
@hunterkepley Do you mean to call this struct `DeleteMachinePoolUserOptions` ? If it can be shared between `describe` and `delete`, how about something like `NameOnlyMachinePoolOptions` or something like that (naming is hard :-) )

Reply:
:+1: may as well reuse, thanks